Add 2022-12-22 agenda and minutes (#87)
solid/notifications-panel: meetings/2022-12-22.md

>ACTION: eP to create issue discussing need for `channel` defined in the vocab or just in JSON-LD context.

> * RG: We didn't specify how does the request and response work when you want to modify the subscription
> * SC: Not in this PR, we still didn't reach consensus with this feature. We might need an issue for it.

Reviewer: ```suggestion
* RG: We didn't specify — how does the request and response work when you want to modify the subscription?
* SC: Not in this PR; we still didn't reach consensus with this feature. We might need an issue for it.
```

* RG: I think we have an issue for it.

> * eP: is there something problematic in 135 if we don't include updating now?

Reviewer: ```suggestion
* eP: is there something problematic in [135](https://github.com/solid/notifications/pull/135) if we don't include updating now?
```

> * RG: No but it will have to do with the data model (eventually). I don't see a problem as another thing but it should be part of the next release.

Reviewer: ```suggestion
* RG: No, but it will have to deal with the data model (eventually). I don't see a problem as another thing, but it should be part of the next release.
```

> * SC: That feature would require both an update to the protocol and data model.
> * eP: RG, do you have a specific proposal? We can take care of this PR and then create issue to work together towards a PR for updating the subscription. Still need a clear approach.

Reviewer: ```suggestion
* SC: That feature would require an update to both the protocol and data model.
* eP: RG, do you have a specific proposal? We can take care of this PR and then create issues to work together toward a PR for updating the subscription. Still need a clear approach.
```

ACTION: RG to check existing issues or create an issue to track updating subscriptions.
RG: https://github.com/solid/notifications/issues/103

> * SC: I tried to incorporate all of the comments. I think there are 2 or 3 which I still didn't get to.

Reviewer: ```suggestion
* SC: I tried to incorporate all the comments. I think there are 2 or 3 which I still didn't get to.
```

* ...: Pavlik suggested we could get rid of the terminology.

> * eP: In general, terminology makes sense if there is no dedicated section and a term used throughout the spec. On the contrary all the terms we have in terminology they have their own section. In that case those sections re normative and we can reuse. It could clean-up linking.

Reviewer: ```suggestion
* eP: In general, terminology makes sense if there is no dedicated section and a term is used throughout the spec. On the contrary, all the terms we have in terminology have their own section. In that case, those sections are normative, and we can reuse them. It could clean up linking.
```

* SC: I suggest we walk through each term and see. I'm +0.75 to removing if they can belong elsewhere.
* SC: Let's dive into it using my screen share
* SC: There is also the issue of clarifying when the subscription step is needed
* eP: I'm ok to have it as dedicated PR

> * SC: We can still try to get it in 135
> * SC: I also pointed out possibly collision of startTime with AS JSON-LD context mapping

Reviewer: ```suggestion
* SC: We can still try to get it in [135](https://github.com/solid/notifications/pull/135)
* SC: I also pointed out possible collision of `startTime` with `AS JSON-LD` context mapping
```

ACTION: Issue for https://github.com/solid/notifications/pull/135#issuecomment-1361127017

* eP: https://www.w3.org/TR/2012/REC-owl2-primer-20121211/#Property_Characteristics

> > Note that this statement does not require every individual to have a husband, it just states that there can be no more than one. Moreover, if we additionally had a statement that Mary's husband is James and another that Mary's husband is Jim, it could be inferred that Jim and James must refer to the same individual.

Reviewer: *sigh* The OWL Primer needs some revision. Among other things, inference that "Jim and James ... refer to the same individual" relies on singular marriage — both that Mary never divorced Jim or James and later married the other, and that Mary could not be married to both Jim and James at the same time. We should try not to perpetuate these (or other) societal projections.

> * eP: If channel can only have one startTime. If there are more than one, it would be invalid. In practice, functional property definition, two start times entails they are the same. Not sure if we can use FunctionalObjectProperty and DataTypeProperty together ... It is safe for now and have separate issue. I think it is wrong definition in the AS2 vocab.
> * SC: Maybe we consider using start/endDateTime in notifications JSON-LD context.

Reviewer: ```suggestion
* eP: Channel can only have one `startTime`. If there is more than one, it would be invalid. In practice, functional property definition: two `startTime`s entails they are the same. Not sure if we can use `FunctionalObjectProperty` and `DataTypeProperty` together ... It is safe for now and have separate issue. I think it is wrong definition in the AS2 vocab.
* SC: Maybe we consider using `startDateTime`/`endDateTime` in notifications JSON-LD context.
```

* SC: It only may collide in the case of notification data model because AS2 JSON-LD Context is not expected to be used in the other data model serializations.
* eP: We can have our own start/end and no collisions. if so, we can resolve later.
* SC: We have this requirement of sending notification if state changed during the request
* eP: IMO we can leave it up to the client to re-check the state after the subscription response
* RG: The client gets burdened with the race condition.
* eP: I think it's easy with a single topic, it may differ with 50 or more topics

> * RG: This is my use case, I watch the folder and all what it contains.

Reviewer: ```suggestion
* RG: This is my use case, I watch the folder and all that it contains.
```

* eP: I think we should create an issue and in the end go with what will come out out of implementation experience.

ACTION: Create issue for `state`. Mentioned it is currently designed for single resource and doesn't handle multiple topics.

### Rename Subscription Resource to Subscription Service?

> RG: I noticed #63 by Cristoph Braun who want to rename Subscription Resource to Subscription Service.

Reviewer: ```suggestion
RG: I noticed #63 by Cristoph Braun who want to rename `Subscription Resource` to `Subscription Service`.
```
